cli/command/image/save_test.go

```diff
@@ -43,6 +43,11 @@ func TestNewSaveCommandErrors(t *testing.T) {
 				return ioutil.NopCloser(strings.NewReader("")), errors.Errorf("error saving image")
 			},
 		},
+		{
+			name:          "output directory does not exist",
+			args:          []string{"-o", "fakedir/out.tar", "arg1"},
+			expectedError: "failed to save image: directory does not exist",
```

Comment:
I guess the error message should also include the word "output", otherwise it may not be clear.